### DISCUSSION TITLE
App is not showing in Unknow sources

### DISCUSSION THREAD

**Discussion by ratozumbi** [icosa-gallery/open-brush#361] [Q&A] [answered]
When installing using adb it says installation successful but nothing shows up on Unkown Sources tab

IDK if this is relevant, but I had the app lab version installed. I uninstalled prior to installing the latest release ([1.0.209](https://github.com/icosa-gallery/open-brush/releases/tag/1.0.209))

My Quest 1 is at version 47 with no updated available

**Comment by mikeage** [comment, marked as answer]
If you've "purchased" the app from app lab (I know it's free, but Oculus considers it a purchase), and then uninstall the app and install from an apk via SideQuest or `adb` directly, it will show up in the main list of apps, just like you downloaded it from the official store / app lab. This is true for all Quest apps.

If you install one of the builds from a PR / branch, then it will have a unique build identifier and appear in Unknown Sources, but this is not your case.

If your goal was to get the latest and greatest build, you can do that through the official Quest system as well; see https://docs.openbrush.app/alternate-and-experimental-builds/open-brush-beta-docs#_lnka1z4mzei7-3 for more details. This also has the advantage of automatically updating.
